src/internal_events/ansi_stripper.rs

```diff
@@ -12,8 +14,8 @@ impl InternalEvent for AnsiStripperFieldMissingError<'_> {
         debug!(
             message = "Field does not exist.",
             field = %self.field,
-            error = "Field does not exist.",
-            error_type = "field_missing",
+            error_code = FIELD_MISSING,
```

Comment:
What do you think about collecting all the `error_code` constants into a module like `error_type` (in a separate PR of course)?